models/public/device_support.md

```diff
@@ -4,106 +4,106 @@
 |------------|-----|-----|--------|
 | aclnet | YES | YES |    |
 | aclnet-int8 | YES | YES |    |
-| alexnet | YES |  | YES |
+| alexnet | YES | YES | YES |
 | anti-spoof-mn3 | YES |  | YES |
 | background-matting-mobilenetv2 | YES |    |    |
-| bert-base-ner | YES |  |    |
+| bert-base-ner | YES | YES |    |
 | brain-tumor-segmentation-0001 | YES |    |    |
 | brain-tumor-segmentation-0002 | YES |    |    |
 | caffenet | YES | YES | YES |
 | cocosnet | YES | YES |    |
 | colorization-siggraph | YES | YES | YES |
 | colorization-v2 | YES | YES | YES |
 | common-sign-language-0001 | YES | YES |    |
-| ctdet_coco_dlav0_512 | YES |  | YES |
-| ctpn | YES |  | YES |
-| deblurgan-v2 | YES |  | YES |
-| deeplabv3 | YES |  | YES |
+| ctdet_coco_dlav0_512 | YES | YES | YES |
+| ctpn | YES | YES | YES |
+| deblurgan-v2 | YES | YES | YES |
+| deeplabv3 | YES | YES | YES |
 | densenet-121 | YES | YES | YES |
-| densenet-121-tf | YES |  | YES |
+| densenet-121-tf | YES | YES | YES |
 | detr-resnet50 | YES |    |    |
 | dla-34 | YES | YES | YES |
-| drn-d-38 | YES |  |    |
-| efficientdet-d0-tf | YES |  | YES |
-| efficientdet-d1-tf | YES |  | YES |
+| drn-d-38 | YES | YES |    |
+| efficientdet-d0-tf | YES |    | YES |
+| efficientdet-d1-tf | YES |    | YES |
 | efficientnet-b0 | YES |  | YES |
```

Comment:
this one ?

Reply:
efficientdet-d0-tf  & efficientdet-d1-tf failed with stack overflow, is it already solved?

Reply:
> efficientdet not enabled on GPU by this change

it was, and I thought your question was about that, ok.

Reply:
I think `efficientnet-b0` is ok and was GPU off by mistake, but we can't test this model as I see. (there is not test case for it)